resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -292,6 +292,14 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_JOB_PRIORITY = ConfigBuilder("spark.kubernetes.job.priorityClassName")
+    .doc("The name of the queue to which the job is submitted. This info " +
```

Comment:
again the doc talks about `queue` (https://volcano.sh/en/docs/vcjob/#queue) while it should be https://volcano.sh/en/docs/vcjob/#priorityclassname